src/light-box/share.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+import { Share } from 'react-native';
+
+import { BRAND_COLOR } from '../styles';
+
+export default (url: string) => {
+  const shareOptions = {
+    message: url,
+    title: 'Shared using Zulip',
```

Comment:
I think we need some better workflow for that since `master` will technically be broken before translating this. (or not in that case?)